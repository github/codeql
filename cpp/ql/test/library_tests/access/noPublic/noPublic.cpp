// Some of our libraries depend on the resulting access of a hypothetical
// public member. This test verifies that the database contains QL
// `AccessSpecifier` instances for both `private` and `public` even when
// extracting a program that does not contain those keywords. Indeed, this
// program contains nothing at all except this comment.
