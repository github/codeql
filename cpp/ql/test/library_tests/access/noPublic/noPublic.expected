| file://:0:0:0:0 | private |
| file://:0:0:0:0 | public |
